src/jdk.jfr/share/classes/jdk/jfr/events/ContainerMemoryUsageEvent.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, DataDog. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.jfr.events;
+
+import jdk.jfr.Category;
+import jdk.jfr.DataAmount;
+import jdk.jfr.Description;
+import jdk.jfr.Enabled;
+import jdk.jfr.Label;
+import jdk.jfr.Name;
+import jdk.jfr.Period;
+import jdk.jfr.StackTrace;
+import jdk.jfr.Threshold;
+import jdk.jfr.Timespan;
+import jdk.jfr.internal.Type;
+
+@Name(Type.EVENT_NAME_PREFIX + "ContainerMemoryUsage")
+@Label("Container Memory Usage")
+@Category({"Operating System", "Container", "Memory"})
+@Description("Container memory usage related information.")
+public final class ContainerMemoryUsageEvent extends AbstractJDKEvent {
+    @Label("Memory Pressure")
+    @Description("(attempts per second * 1000), if enabled, that the operating system tries to satisfy a memory request for any " +
+                 "process in the current container when no free memory is readily available.")
+    public double memoryPressure;
```

Comment:
Should this `memoryPressure` field go from `ContainerMemoryUsageEvent` class? It's not set anywhere is it? would be cgroup v1 only api so I'm not sure it should be there for a generic event like this.

Reply:
Yes. Removing.